com.github.sakya.corechess.json

```diff
@@ -0,0 +1,47 @@
+{
+  "app-id": "com.github.sakya.corechess",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet6"
+  ],
+  "command": "CoreChess",
+  "modules": [
+    {
+      "name": "CoreChess",
+      "buildsystem": "simple",
+      "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet6/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet6/lib",
+        "env": {
+          "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+        }
+      },
+      "build-commands": [
+        "dotnet publish CoreChess.sln -c Release --source ./nuget-sources --runtime linux-x64 -p:PublishReadyToRun=true --self-contained --output ./dist/linux-x64",
+        "cp CoreChess/Lib/Bass/libbass.so ./dist/linux-x64/",
+        "cp -r --remove-destination ./dist/linux-x64 /app/bin/",
```

Comment:
any chance to actually build this from source so that a `aarch64` be available?

Reply:
Are you talking about libbass? There's an arcm64 build. I can include it...